pandas/tests/series/test_arithmetic.py

```diff
@@ -679,6 +680,38 @@ def test_series_add_aware_naive_raises(self):
         with pytest.raises(Exception, match=msg):
             ser_utc + ser
 
+    def test_series_add_daytime_offset(self):
+        # GH#19211
+        # Ignore PerformanceWarning for this test case
+        warnings.simplefilter(action="ignore", category=pd.errors.PerformanceWarning)
```

Comment:
You can use the pytest.mark.filterwarnings decorator for this

Reply:
I already tried this.
```
@pytest.mark.filterwarnings("ignore:pd.errors.PerformanceWarning")
```
and
```
@pytest.mark.filterwarnings("ignore:PerformanceWarning")
```
do not work.

It seams that pytest.mark.filterwarnings can not be used because PerformanceWarning is not a built-in warning class.

Reply:
I think it should work if you match on the warning message instead of the warning itself

Reply:
Nice! Thanks for your advice.